inscrawler/crawler.py

```diff
@@ -83,7 +83,7 @@ def login(self):
         p_input = browser.find_one('input[name="password"]')
         p_input.send_keys(secret.password)
 
-        login_btn = browser.find_one(".L3NKy")
+        login_btn = browser.find_one('button[class="_acan _aiit _acap _aijb _acas _aj1-"]')
```

Comment:
IG updated the css of this button, please take a look.

Reply:
I updated the class name of the button with the latest one.